Staging

eugenex-lab opened the pull request.

eugenex-lab commented:
great addition

eugenex-lab committed:
Merge pull request #11 from Davidson3556/staging1

Solve Merge Conflict

Davidson3556 committed:
modified:   src/router/index.ts

Davidson3556 committed:
modified:   src/views/protected/AdminModule/CreateUser.vue

1rubiz committed:
dashboard ui

Davidson3556 committed:
Merge pull request #2 from Davidson3556/staging1

Staging1

1rubiz committed:
route change and minor tailwind style

eugenex-lab committed:
Merge pull request #12 from Weeshr-Engineering/dashboard-ui

dashboard ui

eugenex-lab committed:
Merge pull request #7 from Davidson3556/main

fix user and admin

eugenex-lab committed:
router check

eugenex-lab committed:
test

1rubiz committed:
admin list page and admin profile page

1rubiz committed:
pagination

ufyy01 committed:
weeshes page ui

ufyy01 committed:
weeshes page background height updated

eugenex-lab committed:
Merge pull request #16 from Weeshr-Engineering/weeshes-page

weeshes page ui

1rubiz committed:
admin list and admin details mod

1rubiz committed:
admin list and admin details mod

eugenex-lab committed:
Merge branch 'staging' into admin-list

eugenex-lab committed:
Merge pull request #13 from Weeshr-Engineering/admin-list

admin list page and admin profile page

1rubiz committed:
usercount api, userhub skeleton

eugenex-lab committed:
Merge pull request #18 from Weeshr-Engineering/usercount

usercount api, userhub skeleton

1rubiz committed:
disable and enable admins feature

1rubiz committed:
Button color adjustment

1rubiz committed:
Minor mod

1rubiz committed:
admin mod

1rubiz committed:
Merge branch 'mod-admin' of https://github.com/Weeshr-Engineering/weeshr-admin into mod-admin

Davidson3556 committed:
modified:   src/router/index.ts

Davidson3556 committed:
modified:   src/components/SideBar.vue
	new file:   src/views/protected/AdminModule/BankPage.vue
	new file:   src/views/protected/AdminModule/DepotPage.vue
	modified:   src/views/protected/AdminModule/UserHub.vue

Davidson3556 committed:
modified:   src/views/protected/AdminModule/UserHub.vue

Davidson3556 committed:
modified:   src/views/protected/AdminModule/UserDetails.vue

Davidson3556 commented on src/views/protected/AdminModule/UserDetails.vue:
Userdetails page. corrected UI

Davidson3556 committed:
modified:   src/components/ui/tabs/TabsList.vue
	modified:   src/components/ui/tabs/TabsTrigger.vue

Davidson3556 committed:
modified:   src/views/protected/AdminModule/UserDetails.vue

Davidson3556 committed:
modified:   src/components/ui/switch/Switch.vue

Davidson3556 committed:
modified:   src/views/protected/AdminModule/AppUsers.vue

Davidson3556 committed:
modified:   src/views/protected/AdminModule/UserDetails.vue

Davidson3556 committed:
ui development

Davidson3556 committed:
modified:   src/views/protected/AdminModule/UserDetails.vue

eugenex-lab committed:
Merge pull request #22 from Weeshr-Engineering/feat/depot-ui

ui development

Davidson3556 committed:
social_media

Davidson3556 committed:
updated the social media link in dashboardfooter

eugenex-lab committed:
Merge pull request #23 from Weeshr-Engineering/social-media

Social media

ufyy01 committed:
Add new user form UI

1rubiz committed:
update admin data

1rubiz committed:
bug fix

eugenex-lab committed:
Merge pull request #20 from Weeshr-Engineering/mod-admin

disable and enable admins feature

ufyy01 committed:
create user form UI updated

ufyy01 committed:
Updates to phone input

ufyy01 committed:
Update to create user form

ufyy01 committed:
Select element updated

Davidson3556 committed:
this page user activity log

1rubiz committed:
country code integration

ufyy01 committed:
merge conflict...

ufyy01 committed:
Merge changes from staging into user-form

Davidson3556 committed:
new file:   src/components/ui/pagination/PaginationEllipsis.vue
	new file:   src/components/ui/pagination/PaginationFirst.vue
	new file:   src/components/ui/pagination/PaginationLast.vue
	new file:   src/components/ui/pagination/PaginationNext.vue
	new file:   src/components/ui/pagination/PaginationPrev.vue
	new file:   src/components/ui/pagination/index.ts
	modified:   src/views/protected/AdminModule/ActivityLog.vue

1rubiz committed:
minor update

eugenex-lab committed:
Merge pull request #24 from Weeshr-Engineering/user-form

Add new user form UI

1rubiz committed:
min-w-32 added

1rubiz committed:
min-w-32 added

1rubiz committed:
Min width

1rubiz committed:
min-w

eugenex-lab committed:
Merge pull request #26 from Weeshr-Engineering/activity-log

this page user activity log

eugenex-lab committed:
Merge pull request #25 from Weeshr-Engineering/countrycode

country code integration

1rubiz committed:
config page

Davidson3556 committed:
UI development and cleanup. removed the permission tab

eugenex-lab committed:
Merge pull request #28 from Weeshr-Engineering/admindetails

UI development and cleanup. removed the permission tab

1rubiz committed:
role manager

1rubiz committed:
role create, delete, edit

Davidson3556 committed:
Detailed Activity Log

Davidson3556 committed:
Detailed Activity Log

Davidson3556 committed:
Detailed and filterd  Activity Log

Davidson3556 committed:
UI adjustment

ufyy01 committed:
user list, create user and get user implemented

eugenex-lab committed:
Merge pull request #29 from Weeshr-Engineering/weeshr-activity-log

Detailed Activity Log

1rubiz committed:
set up edit role

eugenex-lab committed:
Merge pull request #27 from Weeshr-Engineering/role-manager

config page

ufyy01 committed:
usersTable created as a component.

ufyy01 committed:
clear filter now clears search output

eugenex-lab committed:
Merge pull request #31 from Weeshr-Engineering/appUser

user list, create user and get user implemented

1rubiz committed:
updated create admin roles, modify admin roles

1rubiz committed:
test

Davidson3556 committed:
Detailed and filterd  Activity Log WITH MODAL

Davidson3556 committed:
Detailed and filterd  Activity Log WITH MODAL

1rubiz committed:
weesh category

1rubiz committed:
pagination, list users

1rubiz committed:
fix patch api

ufyy01 committed:
User list and profile updated

1rubiz committed:
dob input width

ufyy01 committed:
Form height and switch updated

ufyy01 committed:
Update to pagination

Davidson3556 committed:
Detailed and filterd  Activity Log WITH MODAL PART2

eugenex-lab committed:
Merge pull request #34 from Weeshr-Engineering/user-details

User list and profile updated

eugenex-lab committed:
Merge pull request #32 from Weeshr-Engineering/admin-role

Admin role

Davidson3556 committed:
Depot typo corrections

eugenex-lab committed:
Merge pull request #35 from Weeshr-Engineering/feat/depot-ui

Depot typo corrections and UI cleanup

eugenex-lab committed:
Merge pull request #33 from Weeshr-Engineering/weeshr-activity-log

Detailed and filterd  Activity Log WITH MODAL

1rubiz committed:
CASL permissions: casl setup, admin page integration

Davidson3556 committed:
fix the paginationlist bug

1rubiz committed:
local to session storage

eugenex-lab committed:
Merge pull request #37 from Weeshr-Engineering/weeshr-activity-log

fix the paginationlist bug

1rubiz committed:
remove reload trigger

1rubiz committed:
new data issue

ufyy01 committed:
Weeshes List

ufyy01 committed:
Pagination updated

ufyy01 committed:
UserDetails updated

eugenex-lab committed:
Merge pull request #38 from Weeshr-Engineering/weeshes-list

Weeshes list

ufyy01 committed:
Updated UserWeeshes

eugenex-lab committed:
Merge pull request #39 from Weeshr-Engineering/weeshes-update

Updated UserWeeshes

1rubiz committed:
set abilities for config hub, role manager, weesh category

1rubiz committed:
update admin page and admin details abilities

1rubiz committed:
edit role

1rubiz committed:
err handling for role-manager, weesh-categories, adminlist

Davidson3556 committed:
Implement the New Notification

Davidson3556 committed:
Implement the New Notification

dependabot[bot] committed:
Bump ws from 8.16.0 to 8.17.1

Bumps [ws](https://github.com/websockets/ws) from 8.16.0 to 8.17.1.
- [Release notes](https://github.com/websockets/ws/releases)
- [Commits](https://github.com/websockets/ws/compare/8.16.0...8.17.1)

---
updated-dependencies:
- dependency-name: ws
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

eugenex-lab committed:
Merge pull request #41 from Weeshr-Engineering/dependabot/npm_and_yarn/ws-8.17.1

Bump ws from 8.16.0 to 8.17.1

Davidson3556 committed:
Implement the New Notification and making changes to all the files without variant

Davidson3556 committed:
New and updated Errorpage

eugenex-lab committed:
Merge pull request #40 from Weeshr-Engineering/notification

Implement the New Notification

ufyy01 committed:
User details updated

ufyy01 committed:
Merge branch 'staging' of https://github.com/Weeshr-Engineering/weeshr-admin into userdets-update

1rubiz committed:
disabled features

1rubiz committed:
fixed sidebar permissions and active state

1rubiz committed:
clear local and session storage

ufyy01 committed:
UserDetails updated

eugenex-lab committed:
Merge branch 'staging' into permissions

eugenex-lab committed:
Merge pull request #36 from Weeshr-Engineering/permissions

CASL permissions: casl setup, admin page integration

Davidson3556 committed:
errorpage development

codeDeeAi committed:
[REM] removed user credentials logged in console in signin/auth page

eugenex-lab committed:
Merge pull request #44 from Weeshr-Engineering/errorpage

Errorpage

codeDeeAi committed:
[ADD] API service, 401 interceptor, env.example, API Base url,  [MOD] modified the login function and updated pages to extend new axios service

eugenex-lab committed:
Merge pull request #45 from Weeshr-Engineering/feature/axios-service

[ADD] API service, 401 interceptor, env.example, API Base url,  [MOD]…

codeDeeAi committed:
[BUGFIX] fixed redirect issue on login function

codeDeeAi committed:
[BUGFIX/WIP] fixing redirect issue on login function

codeDeeAi committed:
[ADD] API service request interceptor, [MOD] updated all axios calls to reflect the new req interceptor

codeDeeAi committed:
[ADD] added immutable for roles

codeDeeAi committed:
[ADD] filters to activity logs

codeDeeAi committed:
[ADD] added pagination for activity logs and its filters

codeDeeAi committed:
[REM] removed console.log afrom special pagination component

codeDeeAi committed:
[REM] removed previous handle page change func in activity logs page

codeDeeAi committed:
[MOD] updated the default page number for activity logs

codeDeeAi committed:
[ADD] loading toasts to activity logs

ufyy01 committed:
User wallet updated

1rubiz committed:
weesh details ui

ufyy01 committed:
UserDetails updated

ufyy01 committed:
More updates to user details

1rubiz committed:
integrate weesh list api in users Weesh page

1rubiz committed:
permission for weeshes

codeDeeAi committed:
[ADD] added design for notification board

codeDeeAi committed:
[WIP] integrating notifications api

codeDeeAi committed:
[WIP] implementing message components

codeDeeAi committed:
[UPD] updated priority badge component

codeDeeAi committed:
[MOD] updated interface for Notification Board

codeDeeAi committed:
[ADD] added ReadByList component

codeDeeAi committed:
[ADD] added datefns library

codeDeeAi committed:
[WIP]

codeDeeAi committed:
[ADD] mark as read

codeDeeAi committed:
[ADD] added general notifications data

codeDeeAi committed:
[ADD] added pagination

ufyy01 committed:
error messages updated

eugenex-lab committed:
Merge branch 'staging' into userdets-update

eugenex-lab committed:
Merge pull request #43 from Weeshr-Engineering/userdets-update

User details updated

1rubiz committed:
verification progress

1rubiz committed:
Merge branch 'staging' into weesh-details

Davidson3556 committed:
Implement the dashboard analytics

Davidson3556 committed:
Implement the dashboard analytics

Davidson3556 committed:
Implement the dashboard analytics

Davidson3556 committed:
Implement the dashboard analytics

Davidson3556 committed:
Implement the dashboard analytics and the bg color

1rubiz committed:
home

1rubiz committed:
Merge branch 'weesh-details' of https://github.com/Weeshr-Engineering/weeshr-admin into weesh-details

1rubiz committed:
nav

1rubiz committed:
nav

codeDeeAi committed:
[ADD] timestamp as default sort_column for activity logs

codeDeeAi committed:
[ADD] added delete notifications to admin notifications board

codeDeeAi committed:
[ADD] added open modal func to view button in notification board

Davidson3556 committed:
Implement the dashboard analytics, the bg color and added coma to the data number fetched

Davidson3556 committed:
Implement the dashboard analytics, the bg color and added coma to the data number fetched

1rubiz committed:
ui update

1rubiz committed:
minor scale

1rubiz committed:
Update TabOne.vue

1rubiz committed:
ui fix

Davidson3556 committed:
corrected the foreground

1rubiz committed:
min-h for success tab

eugenex-lab committed:
Merge pull request #47 from Weeshr-Engineering/dashboard-analytics

Dashboard analytics

eugenex-lab committed:
Merge pull request #46 from Weeshr-Engineering/weesh-details

Weesh details

1rubiz committed:
activity and users permission (read & create)

eugenex-lab committed:
Merge pull request #48 from Weeshr-Engineering/activity-permission

activity and users permission (read & create)

codeDeeAi committed:
[ADD] added change to first page when per page values are updated in custom pagination component

codeDeeAi committed:
[TEST] donation progress from api

1rubiz committed:
weesh details api integration

ufyy01 committed:
activity log updated

ufyy01 committed:
Merge branch 'staging' of https://github.com/Weeshr-Engineering/weeshr-admin into user-activity-log

ufyy01 committed:
User activity log filters updated

eugenex-lab committed:
Merge pull request #51 from Weeshr-Engineering/user-activity-log

User activity log

1rubiz committed:
link fix

codeDeeAi committed:
[FIX] notification board pagination

1rubiz committed:
login and weesh categories bug fixes

1rubiz committed:
error toast fix

1rubiz committed:
bug fix list completed

eugenex-lab committed:
Merge pull request #52 from Weeshr-Engineering/login-weesh-fix

Login weesh fix

eugenex-lab committed:
Merge pull request #50 from Weeshr-Engineering/weesh-detail-api

weesh details api integration

1rubiz committed:
edit ability checker

1rubiz committed:
bug fixes

1rubiz committed:
notification

1rubiz committed:
Update PageFilters.vue

1rubiz committed:
Mobile style fix

1rubiz committed:
Update NotificationList.vue

1rubiz committed:
Update NotificationBoardSheet.vue

ufyy01 committed:
Admin Dropdown updated

1rubiz committed:
font-outfit, fix config disable

1rubiz committed:
Merge branch 'err-display-fix' of https://github.com/Weeshr-Engineering/weeshr-admin into err-display-fix

eugenex-lab committed:
Merge pull request #54 from Weeshr-Engineering/err-display-fix

edit ability checker

ufyy01 committed:
Admin nav menu removed

ufyy01 committed:
Merge branch 'staging' of https://github.com/Weeshr-Engineering/weeshr-admin into bug-fix

eugenex-lab committed:
Merge pull request #55 from Weeshr-Engineering/bug-fix

Admin Dropdown updated

1rubiz committed:
payout page ui

1rubiz committed:
permissions

ufyy01 committed:
Payment Approval added

1rubiz committed:
Merge remote-tracking branch 'origin/paymentApproval to use payout
component' into payout

1rubiz committed:
lien component

1rubiz committed:
permissions

1rubiz committed:
sum fix

1rubiz committed:
payout api integration

1rubiz committed:
table min-h

eugenex-lab committed:
Merge pull request #56 from Weeshr-Engineering/payout

payout page ui

1rubiz committed:
featured moment

1rubiz committed:
featured moment

1rubiz committed:
fix edit feature

eugenex-lab committed:
Merge pull request #58 from Weeshr-Engineering/featured

Featured moment

ufyy01 committed:
Bank details

1rubiz committed:
condition to check if scheduled date was created

1rubiz committed:
remove extra loading from weesh config

1rubiz committed:
remove extra loading from weesh config

1rubiz committed:
remove extra loading from weesh config

eugenex-lab committed:
Merge pull request #59 from Weeshr-Engineering/mod-featured

condition to check if scheduled date was created

ufyy01 committed:
bank details updated

ufyy01 committed:
Merge branch 'staging' of https://github.com/Weeshr-Engineering/weeshr-admin into bankDets

1rubiz committed:
outflow and balance api integration

1rubiz committed:
overflow fix

1rubiz committed:
overflow fix

ufyy01 committed:
Bank details updated

eugenex-lab committed:
Merge pull request #61 from Weeshr-Engineering/transfer-api

outflow and balance api integration

ufyy01 committed:
Merge branch 'staging' of https://github.com/Weeshr-Engineering/weeshr-admin into bankDets

ufyy01 committed:
merge confilt fixed

eugenex-lab committed:
Merge pull request #60 from Weeshr-Engineering/bankDets

Bank dets

1rubiz committed:
dashboard : balance loading fix

eugenex-lab committed:
Merge pull request #62 from Weeshr-Engineering/balance_fix

dashboard : balance loading fix

1rubiz committed:
fix bank permission, user creation form, ui to add user to moment

1rubiz committed:
fix bank permission, user creation form, ui to add user to moment

eugenex-lab committed:
Merge pull request #63 from Weeshr-Engineering/fix-user-bank

Fix user bank
